Configuration/StandardSequences/python/ReconstructionCosmics_cff.py

```diff
@@ -44,39 +46,61 @@
 
 # local reco
 trackerCosmicsTask = cms.Task(offlineBeamSpot,trackerlocalrecoTask,MeasurementTrackerEvent,tracksP5Task)
+
+# ugly hack: for the time being remove tracking (no Cosmics seeding in Phase-2) 
+from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
+phase2_tracker.toReplaceWith(trackerCosmicsTask,trackerCosmicsTask.copyAndExclude([tracksP5Task]))
```

Comment:
@mmusich 
do you happen to have a list of things not working from this `tracksP5Task` task?

Reply:
@slava77 

>  do you happen to have a list of things not working from this tracksP5Task task

sort of everything in the seeding part:
   * the seeding layer lists contain layers that don't exist in the phase-2 setup ;
   * the rechit collections used for the outer tracker are `siStripMatchedRecHits` and not `siPhaseRechits`; 
   * the OT hit extractor does not support phase2 rechits (only vector hits)
   * etc. etc.